common/core/overlaylayer.cpp

```diff
@@ -263,6 +255,15 @@ void OverlayLayer::InitializeState(HwcLayer* layer,
     }
   }
 
+ // In case of layer using blening we need to force partial clear. Otherwise
```

Comment:
Also, while you didn't write the comment, just moved it, the misspelling should be fixed.
/blening/blending/

Reply:
After the rotation is done, the source layer dimensions are already updated, so why do we need to do swap again?